src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -235,16 +235,14 @@ def addRuntimeMonitors(self, task):
 
         monitoring = task.data.section_("watchdog")
         monitoring.interval = 300
-        monitoring.monitors = ["DashboardMonitor", "PerformanceMonitor"]
-        monitoring.section_("DashboardMonitor")
-        monitoring.DashboardMonitor.destinationHost = self.dashboardHost
-        monitoring.DashboardMonitor.destinationPort = self.dashboardPort
+        monitoring.monitors = ["PerformanceMonitor"]
         monitoring.section_("PerformanceMonitor")
         monitoring.PerformanceMonitor.maxPSS = maxpss
         monitoring.PerformanceMonitor.softTimeout = softTimeout
         monitoring.PerformanceMonitor.hardTimeout = hardTimeout
         return task
 
+    # FIXME: this function is getting deprecated. Will be removed in March 2020
```

Comment:
Which is the change on March that will deprecate this function? If we need to keep it until then let's make an issue to remind us to remove it when the time comes.

Reply:
It's your turn to open a GH issue :)

Reply:
And here it is :) 

https://github.com/dmwm/WMCore/issues/9513